### ISSUE TITLE
(node:7864) UnhandledPromiseRejectionWarning: Error: invalid value (typically too big) for the size of the input (surface, pattern, etc.)

### ISSUE THREAD

**Issue by kazuiii** [Dalvii/plex-library-export#1]
````
kazui@MBP-kazui plex-library-export-main 2 % node main.js
Films
Script by Dalvi https://github.com/Dalvii/
Il y a 2016 éléments en tout
page 1 avec 2016 images...
(node:7864) UnhandledPromiseRejectionWarning: Error: invalid value (typically too big) for the size of the input (surface, pattern, etc.)
    at Object.createCanvas (/Users/kazui/Downloads/plex-library-export-main 2/node_modules/canvas/index.js:15:10)
    at get (/Users/kazui/Downloads/plex-library-export-main 2/main.js:53:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:7864) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:7864) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
````

i have a library of 2k movies, maybe it's the main reason...
any ideas?

**Comment by kubiksamek** [comment]
This is caused by canvas limit of maximum image size (32768 x 32768px). This can be fixed with this #4 .

**Comment by kubiksamek** [comment]
@kazuiii You can try it again after pulling new changes.
